toolkit/resources/imageconfigs/additionalfiles/iso_initrd/root/runliveinstaller

```diff
@@ -96,6 +96,13 @@ if [[ ! -f "$UNATTENDED_CONFIG_FILE" ]]; then
     # Ensure the userspace speakup connector is up
     systemctl enable espeakup
     systemctl start espeakup
+else # if an unattended iso
+    # if boot type is 'efi'
+    if [[ $(jq ".SystemConfigs[0].BootType" $UNATTENDED_CONFIG_FILE | tr -d '"') == "efi" ]]; then
```

Comment:
I'd suggest to save the string returned by `jq` into a variable, so its name can make it easier to understand what we're comparing here.